packages/js/product-editor/src/components/iframe-editor/editor-canvas.tsx

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Note that this component and `ResizableEditor` both exist under the `edit-site` package in Gutenberg but are not exported.

We should consider making those consumable, but that may require significant changes on their end to make it work for our use case.

Reply:
Maybe we should add a `TODO: ` comment here, or something else explicit, that mentions it being copied from the `edit-site` package.

Reply:
Good idea!  Since you're already AFK today, I'm going to go ahead and merge this though to unblock future work and we can add this comment in one of the following PRs.